src/trainprogram.cpp

```diff
@@ -788,25 +786,9 @@ void trainprogram::scheduler() {
                         &bluetooth::debug);
                 connect(windows_zwift_ocr_thread, &windows_zwift_incline_paddleocr_thread::onInclination, this,
                         &trainprogram::changeInclination);
-                windows_zwift_ocr_thread->start();
-            }
-#endif
-        } else if (settings.value(QZSettings::zwift_workout_ocr, QZSettings::default_zwift_workout_ocr).toBool() &&
-                   bluetoothManager && bluetoothManager->device() &&
-                   (bluetoothManager->device()->deviceType() == bluetoothdevice::TREADMILL ||
```

Comment:
@victorypoint you can create a branch remove the checking of the device type. in this way the inclination change will be emitted also to the bike